src/Components/Create.tsx

```diff
@@ -0,0 +1,163 @@
+      description:
+        'Now you can see the newly added element on the home page.',
+    });
+  };
+
+  const onFinishFailed = (errorInfo: any) => {
+    console.log('Failed:', errorInfo);
+  };
+
+  return (
+
+    <div className="form-input">
+      <div className="form-input-head">
+        <h2>Fill the form to add new journey!</h2>
+      </div>
+      <Form
+        labelCol={{ span: 4 }}
+        wrapperCol={{ span: 14 }}
+        layout="horizontal"
+        onSubmitCapture={handleSubmit}
+        onFinishFailed={onFinishFailed}
+      >
+        <Form.Item label="Title" rules={[{ required: true, message: 'Please add a title!' }]}>
+          <Input value={title} onChange={(e) => setTitle(e.target.value)} />
+        </Form.Item>
+
+        <Form.Item label="Country">
+          <Input value={country} onChange={(e) => setCountry(e.target.value)} />
+        </Form.Item>
+
+        <Form.Item label="Location">
+          <Input
+            value={location}
+            onChange={(e) => setLocation(e.target.value)}
+          />
+        </Form.Item>
+
+  {/*       <Form.Item label="Date">
+          <Input
+            type="number"
+            value={date}
+            onChange={(e) => setDate(e.target.value)}
+          />
+        </Form.Item> */}
+          <Form.Item label="RangePicker">
+          <RangePicker  onChange={(e) => {console.log(e?.[0]?.toDate())
+            setDateRange({
+                start: e?.[0]?.toDate(),
+                end: e?.[1]?.toDate()
+            })}
+        }/>
+
+       
+        </Form.Item> 
+
+        <Form.Item label="Description">
+          <TextArea
+            rows={4}
+            value={description}
+            onChange={(e) => setDescription(e.target.value)}
+          />
+        </Form.Item>
+
+        <Form.Item label="Author">
+          <Input value={author} onChange={(e) => setAuthor(e.target.value)} />
+        </Form.Item>
+
+        <Form.Item label="Travel companions">
+          <InputNumber
+            value={travelCompanions}
+            onChange={(e) => setTravelCompanions(e)}
+          />
+        </Form.Item>
+
+        <Form.Item label="Rating">
+          <span>
+            <Rate tooltips={desc} onChange={setRating} value={rating} />
+            {rating ? <span className="ant-rate-text">{desc[rating - 1]}</span> : ''}
+          </span>
+        
+        </Form.Item>
+
+        <Form.Item label="Upload" valuePropName="fileList">
+          <Upload action="/upload.do" listType="picture-card">
+            <div>
+              <PlusOutlined />
+              <div style={{ marginTop: 8 }}>Upload</div>
+            </div>
+          </Upload>
+        </Form.Item>
+
+        
+          {!isPending && <ButtonDiv><StandardButton htmlType="submit" onClick={openNotification}>Submit</StandardButton></ButtonDiv>}
+          {isPending && <ButtonDiv><StandardButton disabled>Adding your new travel...</StandardButton></ButtonDiv>}
```

Comment:
Nagyon orulok neki, hogy ezt lekezelted. Hatalmas +1.

```
If A ...
If !A ...
```
Helyett lehetne if-else is, picit tisztabb.

Reply:
Vagy esetleg hasznalhatod a [ternary operatort.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) Az talan meg szebb jelen esetben. 🤔